addons/website_forum/controllers/main.py

```diff
@@ -11,6 +11,7 @@
 from datetime import datetime
 
 from odoo import http, tools, _
+from odoo.exceptions import UserError
```

Comment:
Alphabetical order, bellow the `from odoo.add..`
Sorry to be annoying, I'd gladly do it myself to merge this PR but I can't since this is on your repository :)